src/commands/dependabot/consolidate.ts

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as os from 'os';
+import { fs, Messages } from '@salesforce/core';
+import { flags, FlagsConfig, SfdxCommand } from '@salesforce/command';
+import { Octokit } from '@octokit/core';
+import { Env } from '@salesforce/kit';
+import { diff, ReleaseType } from 'semver';
+import { bold, cyan, green } from 'chalk';
+import { exec } from 'shelljs';
+import { ensureString, isString } from '@salesforce/ts-types';
+import { PackageJson } from '../../package';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-release-management', 'dependabot.consolidate');
+
+export type BumpType = Extract<ReleaseType, 'major' | 'minor' | 'patch'>;
+
+export default class Consolidate extends SfdxCommand {
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessage('examples').split(os.EOL);
+  public static readonly flagsConfig: FlagsConfig = {
+    'max-version-bump': flags.enum({
+      description: messages.getMessage('maxVersionBump'),
+      char: 'm',
+      options: ['major', 'minor', 'patch'],
+      default: 'minor',
+      required: true,
+    }),
+    'base-branch': flags.string({
+      description: messages.getMessage('baseBranch'),
+      char: 'b',
+      default: 'main',
+      required: true,
+    }),
+    'target-branch': flags.string({
+      description: messages.getMessage('targetBranch'),
+      char: 't',
+      default: 'consolidate-dependabot',
+      required: true,
+    }),
+    ignore: flags.array({
+      description: messages.getMessage('ignore'),
+      default: [],
+    }),
+    dryrun: flags.boolean({
+      description: messages.getMessage('dryrun'),
+      char: 'd',
+      default: false,
+    }),
+    'no-pr': flags.boolean({
+      description: messages.getMessage('noPR'),
+      default: false,
+    }),
+    owner: flags.string({
+      description: messages.getMessage('owner'),
+      char: 'o',
+    }),
+    repo: flags.string({
+      description: messages.getMessage('repo'),
+      char: 'r',
+    }),
+  };
+
+  public async run(): Promise<void> {
+    const { owner, repo } = await this.getOwnerAndRepo();
+    const baseBranch = ensureString(this.flags['base-branch']);
+    const targetBranch = ensureString(this.flags['target-branch']);
+    const auth = ensureString(new Env().getString('GH_TOKEN'), 'GH_TOKEN is required to be set in the environment');
+
+    const octokit = new Octokit({ auth });
+    const pullRequests = await octokit.request('GET /repos/{owner}/{repo}/pulls', {
+      owner,
+      repo,
+    });
+
+    const dependabotPRs = pullRequests.data.filter(
```

Comment:
how would you want to handle red PRs?  Manually add them to the ignore flag?  Or is there a way to inspect the pulls object (or some child thereof) and see checks states?

maybe controlled by an 'only-green' flag?

Reply:
also maybe we want to check the mergability (has no conflicts with base)

Reply:
@mshanemc I considered adding that but I decided not to because a build could fail for reasons other than failing unit tests (you'll see there's quite a bit of this if you look through all the dependabot PRs)

My goal with the command was just to get the process started - not to automate the process entirely. With that goal in mind, I thought it'd be okay to allow the failing ones in because I can easily remove any bad commits later if I need to